gnowsys-ndf/gnowsys_ndf/settings.py

```diff
@@ -823,6 +823,12 @@
 # following has to be at last
 # just put every thing above it
 
+#*******MIO_SETTINGS*************
+DEFAULT_MIO_FROM_EMAIL = 'ps.mio.bits@gmail.com'
+GSTUDIO_MIO_FROM_EMAIL = ''
+GSTUDIO_MIO_FROM_EMAIL_PASSWORD = '1Guesswhat'
+#*********************************
+
```

Comment:
We have **DEFAULT_FROM_EMAIL** in _settings.py_. That should be used as default mail for MIO too.
What we need will be **GSTUDIO_MIO_FROM_EMAIL** that will be used to override the **DEFAULT_FROM_EMAIL**